frontend/src/lib/components/BillingAlerts.tsx

```diff
@@ -15,29 +14,17 @@ export function BillingAlerts(): JSX.Element | null {
 
     return (
         <>
-            <div style={{ marginTop: '1.5rem' }} />
-            {alertToShow === BillingAlertType.SetupBilling && (
-                <Card>
-                    <div style={{ display: 'flex' }}>
-                        <div style={{ flexGrow: 1, display: 'flex', alignItems: 'center' }}>
-                            <WarningOutlined className="text-warning" style={{ paddingRight: 8 }} />
-                            <b>Action needed!&nbsp;</b>
-                            {billing?.plan?.custom_setup_billing_message ||
-                                'Please finish setting up your billing information.'}
-                        </div>
-                        <div style={{ display: 'flex', alignItems: 'center' }}>
-                            <Button type="primary" href={billing?.subscription_url} icon={<ToolFilled />}>
-                                Set up now
-                            </Button>
-                        </div>
-                    </div>
-                </Card>
-            )}
-            {alertToShow === BillingAlertType.UsageNearLimit && (
-                <Card>
-                    <div style={{ display: 'flex' }}>
-                        <div style={{ flexGrow: 1, display: 'flex', alignItems: 'center' }}>
-                            <WarningOutlined className="text-warning" style={{ paddingRight: 16 }} />
+            <div className="p-6 mt-6 border rounded-lg flex items-center">
```

Comment:
One of the more refactored components. I checked out each style individually by hardcoding the values and it looked good